pkg/adapterManager/manager_test.go

```diff
@@ -60,3 +104,47 @@ func TestManager(t *testing.T) {
 
 	}
 }
+
+func TestRecovery_NewAspect(t *testing.T) {
+	name := "NewAspect Throws"
+	cacheThrow := newTestManager(name, true, false)
+	m := NewManager([]aspect.Manager{cacheThrow})
+	if err := m.Registry().RegisterDenyChecker(cacheThrow); err != nil {
+		t.Errorf("Failed to register deny checker in test setup with err: %v", err)
+	}
+
+	cfg := &aspect.CombinedConfig{
+		&istioconfig.Aspect{Kind: name},
+		&istioconfig.Adapter{Name: name},
+	}
+
+	if _, err := m.Execute(cfg, nil, nil); err != nil {
```

Comment:
might consider re-ordering this:

```
_, err := m.Execute...
if err == nil {
   t.Error("Execute(): expected error")
}
if !strings... {
    t.Errorf("Execute() => got '%v', want message containing 'NewAspect'", err)
}
```

That would eliminate the nested ifs.

Same suggestion for `TestRecovery_AspectExecute`.